frontend/src/App.js

```diff
@@ -1,14 +1,126 @@
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+            <div>"There are two kinds of people.  Those who don't believe in miracles, and those who believe that everything is a miracle.  And I, for one, am the latter."  - Albert Einstein</div>
+        </div>
+    );
+}
+
+const WebsiteScrollbar = () => {
+    return (
+        <div className="-webkit-scrollbar"></div>
+    );
+}
+
+
+class Clock extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { date: new Date() };
+    }
+
+    componentDidMount() {
+        this.timerID = setInterval(
+            () => this.tick(),
+            1000
+        );
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.timerID);
+    }
+
+    tick() {
+        this.setState({
+            date: new Date()
+        });
+    }
+
+    render() {
+        return (
+            <div style={{ backgroundColor: 'black', color: '#ff7979' }}>
+                <h1 style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>Hello, world!</h1>
+                <h2 style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>It is {this.state.date.toLocaleTimeString()}.</h2>
+            </div>
+        );
+    }
+}
+
+
+class Toggle extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { isToggleOn: true };
+
+        // This binding is necessary to make `this` work in the callback
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        this.setState(state => ({
+            isToggleOn: !state.isToggleOn
+        }));
+    }
+
+    render() {
+        return (
+            <div style={{
+                backgroundColor: 'black', display: 'flex', justifyContent: 'center', alignItems: 'center'
+            }}>
+                <button onClick={this.handleClick}>
+                    {this.state.isToggleOn ? 'ON' : 'OFF'}
+                </button>
+            </div>
+        );
+    }
+}
+
```

Comment:
This is pretty cool. but I would suggest factoring them out into another components.

Like I mention in README, `App.js` should only contain thing that relevant to the entire application. the rest should be separated